docs/releases/v2022.1.rst

```diff
@@ -0,0 +1,419 @@
+
+ramips-mt7620
+~~~~~~~~~~~~~
+
+-  Xiaomi
+
+   -  Mi Router 3G (v2)
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+-  Cudy
+
+   -  WR2100
+
+-  Netgear
+
+   -  R6260
+   -  WAC104
+   -  WAX202
+
+-  TP-Link
+
+   -  RE500
+   -  RE650v1
+
+-  Ubiquiti
+
+   -  UniFi 6 Lite
+
+-  Xiaomi
+
+   -  Mi Router 4A (Gigabit Edition)
+
+ramips-mt7622
+~~~~~~~~~~~~~
+
+-  Linksys
+
+   -  E8450
+
+-  Xiaomi
+
+   -  AX3200
+
+-  Ubiquiti
+
+   -  UniFi 6 LR
+
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+-  GL.iNet
+
+   -  microuter-N300
+
+- Netgear
+
+   - R6020
+
+-  RAVPower
+
+   -  RP-WD009
+
+-  TP-Link
+
+   -  Archer C20 v4
+   -  Archer C20 v5
+   -  RE200 v2
+   -  RE305 v1
+
+-  Xiaomi
+
+   -  Mi Router 4C
+   -  Mi Router 4A (100M Edition)
+
+rockchip-armv8
+~~~~~~~~~~~~~~
+
+-  FriendlyElec
+
+   -  NanoPi R2S
+
+mpc85xx-p1010
+~~~~~~~~~~~~~
+
+-  Sophos
+
+   -  RED 15w rev. 1
+
+mpc85xx-p1020
+~~~~~~~~~~~~~
+
+-  Extreme Networks
+
+   -  WS-AP3825i
+
+Removed Devices
+---------------
+
+This list contains devices which do not have enough memory or flash to
+be operated with this Gluon release.
+
+-  D-Link
+
+   -  DIR-615 (C1, D1, D2, D3, D4, H1)
+
+-  Linksys
+
+   -  WRT160NL
+
+-  TP-Link
+
+   -  TL-MR13U (v1)
+   -  TL-MR3020 (v1)
+   -  TL-MR3040 (v1, v2)
+   -  TL-MR3220 (v1, v2)
+   -  TL-MR3420 (v1, v2)
+   -  TL-WA701N/ND (v1, v2)
+   -  TL-WA730RE (v1)
+   -  TL-WA750RE (v1)
+   -  TL-WA801N/ND (v1, v2, v3)
+   -  TL-WA830RE (v1, v2)
+   -  TL-WA850RE (v1)
+   -  TL-WA860RE (v1)
+   -  TL-WA901N/ND (v1, v2, v3, v4, v5)
+   -  TL-WA7210N (v2)
+   -  TL-WA7510N (v1)
+   -  TL-WR703N (v1)
+   -  TL-WR710N (v1, v2)
+   -  TL-WR740N (v1, v3, v4, v5)
+   -  TL-WR741N/ND (v1, v2, v4, v5)
+   -  TL-WR743N/ND (v1, v2)
+   -  TL-WR840N (v2)
+   -  TL-WR841N/ND (v3, v5, v7, v8, v9, v10, v11, v12)
+   -  TL-WR841N/ND (v1, v2)
+   -  TL-WR843N/ND (v1)
+   -  TL-WR940N (v1, v2, v3, v4, v5, v6)
+   -  TL-WR941ND (v2, v3, v4, v5, v6)
+   -  TL-WR1043N/ND (v1)
+   -  WDR4900
+
+-  Ubiquiti
+
+   -  AirGateway
+   -  AirGateway Pro
+   -  AirRouter
+   -  Bullet
+   -  LS-SR71
+   -  Nanostation XM
+   -  Nanostation Loco XM
+   -  Picostation
+
+-  Unknown
+
+   -  A5-V11
+
+-  VoCore
+
+   -  VoCore (8M, 16M)
+
+Atheros target migration
+------------------------
+
+All Atheros MIPS devices built with the ``ar71xx-generic``,
+``ar71xx-nand`` as well as ``ar71xx-tiny`` were deprecated upstream and
+are therefore not available with Gluon anymore.
+
+Many devices previously built with ``ar71xx-generic`` and
+``ar71xx-nand`` are now available with the ``ath79-generic`` as well as
+``ath79-nand`` target respectively.
+
+Missing devices
+~~~~~~~~~~~~~~~
+
+The following devices have not yet been integrated into Gluons ath79
+targets.
+
+-  8Devices
+
+   -  Carambola 2
+
+-  Aerohive
+
+   -  HiveAP 121
+
+-  Allnet
+
+   -  ALL0315
+
+-  Buffalo
+
+   -  WZR-HP-G300NH2
+   -  WZR-HP-G450H
+
+-  GL.iNet
+
+   -  6408A v1
+
+-  NETGEAR
+
+   -  WNDR3800
```

Comment:
Support was just (re)added for WNDR3800
Can be removed from list of not migrated devices.